data/global.json

```diff
@@ -1,5 +1,5 @@
 {
-  "global": {
+  "tokens":{
```

Comment:
can something ELI5 why we need this top level object? i'd love to just nix it all together but typography in particular gets mad

Reply:
I think its due to how Figma tokens is expecting to be formatted, so it can read it properly?

Reply:
i thought this was something we added, when we thought we would be adding light theme within this file also 

![image](https://user-images.githubusercontent.com/50302041/182853140-68ba83a2-ed37-4e49-9807-9c65a1d7306b.png)